packages/@react-spectrum/datepicker/stories/DatePicker.stories.tsx

```diff
@@ -11,84 +11,243 @@
  */
 
 import {action} from '@storybook/addon-actions';
+import {CalendarDate, CalendarDateTime, parseAbsolute, parseAbsoluteToLocal, parseDate, parseDateTime, parseZonedDateTime, toZoned} from '@internationalized/date';
 import {DatePicker} from '../';
+import {Flex} from '@react-spectrum/layout';
+import {Item, Picker, Section} from '@react-spectrum/picker';
+import {Provider} from '@react-spectrum/provider';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
+import {useLocale} from '@react-aria/i18n';
 
 const BlockDecorator = storyFn => <div>{storyFn()}</div>;
 
-storiesOf('DatePicker', module)
+storiesOf('Date and Time/DatePicker', module)
   .addDecorator(BlockDecorator)
   .add(
     'default',
     () => render()
   )
-  .add(
-    'isQuiet',
-    () => render({isQuiet: true})
-  )
   .add(
     'defaultValue',
-    () => render({defaultValue: new Date(2020, 2, 3)})
+    () => render({defaultValue: parseDate('2020-02-03')})
   )
   .add(
     'controlled value',
-    () => render({value: new Date(2020, 2, 3)})
+    () => render({value: new CalendarDate(2020, 2, 3)})
+  )
+  .add(
+    'defaultValue, zoned',
+    () => render({defaultValue: toZoned(parseDate('2020-02-03'), 'America/Los_Angeles')})
+  )
+  .add(
+    'granularity: month',
+    () => render({granularity: 'month'})
+  )
+  .add(
+    'granularity: minute',
+    () => render({granularity: 'minute'})
+  )
+  .add(
+    'granularity: second',
+    () => render({granularity: 'second'})
+  )
+  .add(
+    'hourCycle: 12',
+    () => render({granularity: 'minute', hourCycle: 12})
+  )
+  .add(
+    'hourCycle: 24',
+    () => render({granularity: 'minute', hourCycle: 24})
+  )
+  .add(
+    'granularity: minute, defaultValue',
+    () => render({granularity: 'minute', defaultValue: parseDateTime('2021-03-14T08:45')})
+  )
+  .add(
+    'granularity: minute, defaultValue, zoned',
+    () => render({granularity: 'minute', defaultValue: parseZonedDateTime('2021-11-07T00:45-07:00[America/Los_Angeles]')})
+  )
+  .add('granularity: minute, defaultValue, zoned, absolute',
+    () => render({granularity: 'minute', defaultValue: parseAbsoluteToLocal('2021-11-07T07:45:00Z')})
+  )
+  .add('granularity: minute, defaultValue, zoned, absolute, timeZone',
+    () => render({granularity: 'minute', defaultValue: parseAbsolute('2021-11-07T07:45:00Z', 'America/New_York')})
   )
   .add(
-    'custom date format',
-    () => render({
-      formatOptions: {
-        year: 'numeric',
-        month: 'long',
-        day: 'numeric',
-        hour: 'numeric',
-        minute: 'numeric',
-        second: 'numeric'
-      }
-    })
+    'hideTimeZone',
+    () => render({granularity: 'minute', defaultValue: parseZonedDateTime('2021-11-07T00:45-07:00[America/Los_Angeles]'), hideTimeZone: true})
```

Comment:
What is this story supposed to demonstrate. I see that the hideTimeZone will cause the formatOptions to not return a `timeZoneName` thus making the formatted date not have something like PST/EST/etc but is there something visual that should be different? Or should the onChange date time not return time zone info?

Reply:
It's the same as the zoned story, but hides the timezone abbreviation.